en/user_guide/content.rst

```diff
@@ -0,0 +1,65 @@
+Contents
+========
+
+Contens are instances of a Content Types. A Content Type is the pattern of a family of Contents. If
+"Car" was a Content Type, "Ford mustang" or "Dodge Viper" could be contents of that type.
+See `Content Types`_ documentation for more information.
+
+Contents are managed in the Back Office.
+
+
+Listing
+-------
+Contents listing is accessed in the Editorial section, by clicking on the "Contents" link. A submenu
+lists all Content Types availables.
+
+.. image:: ../../images/content_list_submenu.png
+
+By clicking a Content Type, the list of related contents appears.
+
+.. image:: ../../images/content_list.png
+
+This list has a pagination and filters, and shows the name, the status, the version and the language
+of existing contents. From this list, new content can be created, and existing ones can be edited or
+deleted.
+
+
+Creation
+--------
+From the content list, creation form can be accessed by clicking on the "Add" button. Information asked
+depend on the Content Type: each Content Type has its own form. By clicking on the "Save" button, the
+content is created, with the version number 1 and the initial status of the workflow.
+
+
+Edition
+-------
+From the content list, clicking on an "Edit" button makes an edit form appear. The edit form is mainly
+the same as the creation form, completed with current values, but has three more functionnalities.
+
+.. image:: ../../images/content_edit.png
+
+* **Status**: on the upper right corner of the form, a button allows you to change the workflow
+  status of the content (See the `workflow`_ documentation for more information)
+
+* **Translations**: at the top of the form, the language tabs allow you to create/edit the content in
```

Comment:
My previous point was to suppress the ``at the top of the form`` part. I think that this would be more efficient to not put it or to put a number in the screenshot.

Those description will be very hard to maintain when we update the page